File: WebAppVen/Scripts/app/model/ME_DOCM.js
﻿Ext.define('WEBAPP.model.ME_DOCM', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'DOCNO', type: 'string' },
        { name: 'DOCTYPE', type: 'string' },
        { name: 'FLOWID', type: 'string' },
        { name: 'APPID', type: 'string' },
        { name: 'APPDEPT', type: 'string' },
        { name: 'APPTIME', type: 'string' },
        { name: 'USEID', type: 'string' },
        { name: 'USEDEPT', type: 'string' },
        { name: 'FRWH', type: 'string' },
        { name: 'TOWH', type: 'string' },
        { name: 'LIST_ID', type: 'string' },
        { name: 'APPLY_KIND', type: 'string' },
        { name: 'APPLY_NOTE', type: 'string' },
        { name: 'CREATE_TIME', type: 'string' },
        { name: 'CREATE_USER', type: 'string' },
        { name: 'UPDATE_TIME', type: 'string' },
        { name: 'UPDATE_USER', type: 'string' },
        { name: 'UPDATE_IP', type: 'string' },

        { name: 'FLOWID_N', type: 'string' },
        { name: 'MAT_CLASS_N', type: 'string' },
        { name: 'FRWH_N', type: 'string' },
        { name: 'TOWH_N', type: 'string' },
        { name: 'APPLY_KIND_N', type: 'string' },
        { name: 'APPTIME_T', type: 'string' },

        // =================== 紹朋 ======================
        { name: 'APPTIME_TWN', type: 'string' },
        // ===============================================

        // =================== 湘倫 ======================
        { name: 'APP_NAME', type: 'string' },
        { name: 'APPDEPT_NAME', type: 'string' },
        { name: 'USEDEPT_NAME', type: 'string' },
        { name: 'FRWH_NAME', type: 'string' },
        { name: 'TOWH_NAME', type: 'string' }
        // ===============================================
    ]
});

File: WebApp/Models/AA/ME_DOCM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApp.Models
{
    public class ME_DOCM : JCLib.Mvc.BaseModel
    {
        public string DOCNO { get; set; }
        public string DOCTYPE { get; set; }
        public string FLOWID { get; set; }
        public string APPID { get; set; }
        public string APPNAME { get; set; }         // 自訂 人員的中文姓名
        public string APPDEPT { get; set; }
        public string APPDEPTNAME { get; set; }     // 自訂 部門的中名稱
        public string APPTIME { get; set; }
        public string USEID { get; set; }
        public string USEDEPT { get; set; }
        public string USEDEPTNAME { get; set; }     // 自訂 使用部門的中名稱
        public string FRWH { get; set; }
        public string FRWHNAME { get; set; }        // 自訂 核撥庫房的中名稱
        public string TOWH { get; set; }
        public string TOWHNAME { get; set; }        // 自訂 領用庫房的中名稱
        public string PURCHASENO { get; set; }
        public string SUPPLYNO { get; set; }
        public string APPLY_KIND { get; set; }
        public string MAT_CLASS { get; set; }
        public string APPLY_NOTE { get; set; }
        public string CREATE_DATE { get; set; }
        public string CREATE_USER { get; set; }
        public string UPDATE_DATE { get; set; }
        public string UPDATE_USER { get; set; }
        public string UPDATE_IP { get; set; }
    }
}